docs/graphql/manual/mutations/upsert.rst

```diff
@@ -104,6 +104,53 @@ the columns specified in ``update_columns``:
 
 The ``published_on`` column is left unchanged as it wasn't present in ``update_columns``.
 
+Update selected columns on conflict using a filter
+--------------------------------------------------
+Insert a new object in the ``article`` table or, if the primary key constraint ``article_pkey`` is violated, update
+the columns specified in ``update_columns`` only if provided ``where`` condition is met:
```

Comment:
There should be a comma before the “or” in this sentence, and a “the” between “provided” and “`where` condition.” Generally this sentence reads very nicely, though. 👍